gobblin-modules/gobblin-helix/src/main/java/gobblin/metastore/ZkStateStore.java

```diff
@@ -192,6 +222,25 @@ public T get(String storeName, String tableName, String stateId) throws IOExcept
   }
 
   /**
+   * Get a {@link State} with a given state ID from a the current table.
```

Comment:
I feel like the goal would be to transition all calls to getCurrent/getAllCurrent and deprecate the "current" name.  Would you prefer to stick with the "current" name and not have the more explicit api?